force_wfmanager/ui/review/plot.py

```diff
@@ -228,7 +226,7 @@ def update_value_names(self):
             self._plot.x_axis.title = ""
             self._plot.y_axis.title = ""
```

Comment:
UX issue: This might be being called when the color plot radio button is being selected, since it removes the axes titles.

The titles will only appear again if either `x` or `y` variables are changed